### PR TITLE
Destination postgres: enable in-connector normalization

### LINKED ISSUE #33458: Add in container normalization for postgres

### PR DESCRIPTION
**Pull request by edgao**
closes https://github.com/airbytehq/airbyte/issues/33458. Requires https://github.com/airbytehq/airbyte/pull/33941 to be merged first.

I ran a [test sync](https://cloud.airbyte.com/workspaces/b61bc266-ef3c-460c-af2f-f70da4a2993c/connections/c2f4da3d-f01b-4010-9cb0-08ed0a5fbbdd/job-history#7188626::0) against a prerelease build and it ran normalization within the destination successfully:
image

### CODE REVIEW

File: airbyte-integrations/bases/base-java/run_with_normalization.sh [hunk 1]
```diff
@@ -36,6 +36,10 @@ then
 elif test "$NORMALIZATION_TECHNIQUE" = 'LEGACY' && test "$USE_1S1T_FORMAT" != "true"
 then
   echo '{"type": "LOG","log":{"level":"INFO","message":"Starting in-connector normalization"}}'
+  # Normalization tries to create this file from the connector config and crashes if it already exists
+  # so just nuke it and let normalization recreate it.
+  # Use -f to avoid error if it doesn't exist, since it's only created for certain SSL modes.
+  rm -f ca.crt
```
Comment (evantahler) [inline airbyte-integrations/bases/base-java/run_with_normalization.sh:42]: Insanity

**Review by evantahler — APPROVED** [review]
I guess AirbyteCI does all the heavy lifting now!

👍 assuming you ran some test syncs

File: airbyte-integrations/bases/base-java/run_with_normalization.sh (hunk 1 above)
Comment (edgao) [reply airbyte-integrations/bases/base-java/run_with_normalization.sh:42]: to be fair, destination-postgres is also creating this file (which is why this is only becoming a problem now) https://github.com/airbytehq/airbyte/blob/2d79854279fdf38d70776b57bc003ec9e6a9119e/airbyte-cdk/java/airbyte-cdk/core/src/main/java/io/airbyte/cdk/integrations/util/PostgresSslConnectionUtils.java#L139

not that I understand why we need to write a file to begin with :shrug:

**Comment by edgao** [comment]
.... local OSS sync worked fine; [cloud sync](https://cloud.airbyte.com/workspaces/b61bc266-ef3c-460c-af2f-f70da4a2993c/connections/c2f4da3d-f01b-4010-9cb0-08ed0a5fbbdd/job-history#7185141::0) didn't run normalization. Proooobably a strict-encrypt problem 🤦 

time to go spelunking in airbyte-ci code I guess

**Comment by edgao** [comment]
or the prerelease build got messed up? `0.5.0-dev.bc7dba0bb1` correctly has run_with_normalization.sh as the entrypoint; `0.5.0-dev.7cbb40e7e4` just has base.sh

... this feels eerily similar to that dagger caching thing we had a few months ago

**Comment by edgao** [comment]
[sync](https://cloud.airbyte.com/workspaces/b61bc266-ef3c-460c-af2f-f70da4a2993c/connections/c2f4da3d-f01b-4010-9cb0-08ed0a5fbbdd/job-history#7188626::0) is still running b/c our instance is small and our data is large, but normalization is in-flight!

image

Holding off on release until we sort out the dagger caching stuff though.

**Pull request merged**